server/src/main/java/org/candlepin/model/ManifestFileRecordType.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2009 - 2012 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.model;
+
+/**
+ * Defines the manifest type of a stored file record.
+ */
+public enum ManifestFileRecordType {
```

Comment:
This type was intended to be passed as part of the service calls but was initially part of the model object directly. I've renamed the enum and moved some other associated files to the appropriate package. Does this work?